lang/cs/Org.Apache.REEF.Examples.HelloREEF/Readme.md

```diff
@@ -6,4 +6,25 @@ This project contains a simple example of a REEF Program. It contains the follow
   * `HelloTask`: This Task prints a greeting to STDOUT of the Evaluator.
 
 ## Running it
-Just run the main class, `HelloREEF`, followed by the runtime you want, e.g. `local`.
\ No newline at end of file
+Just run the main class, `HelloREEF`, followed by the runtime you want, e.g. `local`.
```

Comment:
Interesting. I'll update this doc once REEF-1841 is in. I realized that I forgot 2 steps.